.vib/solr/goss/goss-wait.yaml

```diff
@@ -0,0 +1,9 @@
+addr:
```

Comment:
Could you add a comment about why this sleep is needed? I understand this is needed for the "proper" tests to execute successfully because something is not yet ready when the goss action is executed, but it seems tricky. If there is a component not ready when the deployment action finishes, then that would be an issue with the chart's probes design, WDYT?

Reply:
Yes, I totally agree that this should be something to look at on the chart's side. However, I don't consider this as something blocking the test addition.

I think that I can create a new issue to evaluate it and add the comment on the `goss-wait.yaml` file as you suggested. Do you agree?

Reply:
I agree we shouldn't block this PR, let's do that and revisit this after the probes' logic is optimised.

Reply:
Cool, I added a comment in the file as suggested 🙂